cmd/snap-confine/snap-device-helper

```diff
@@ -18,6 +18,12 @@ MAJMIN="$4"
 APPNAME="$( echo "$APPNAME" | tr '_' '.' )"
 app_dev_cgroup="/sys/fs/cgroup/devices/$APPNAME"
 
+# The tag may be assigned before cgroup is constructed so don't do anything
+# if the croup doesn't exist. LP: #1762182
```

Comment:
Rephrasing this might make it slighly more clear:
```
# The cgroup is only present on snap start so ignore any cgroup changes
# (eg, 'add' on boot, hotplug, hotunplug) when the cgroup doesn't exist
# yet. LP: #1762182.
```

Reply:
Updated and force-pushed for the back port.